src/gui/DisplayPanel.h

```diff
@@ -77,6 +77,7 @@ class
     DisplayNotebook*      my_notebook;
     wxStaticText*         StatusText;
     bool                  popup_exists;
+    bool                  is_from_display_program = false;
```

Comment:
Minor point, but in this case, you might prefer:

```cpp
bool is_from_display_program{};
//or
bool is_from_display_program{false};
```

See some points and discussions in the [cpp core guidlines.](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es23-prefer-the--initializer-syntax)